griptape/tools/__init__.py

```diff
@@ -44,4 +45,5 @@
     "Computer",
     "ProxycurlClient",
     "OpenWeatherClient",
+    "ImageGenerator",
```

Comment:
I can imagine future support for other image generation providers beyond DALL-E. Can you rename this tool to something like DalleImageGenerator?

Reply:
Hey Andrew, got it! I just realized I haven't used error artifacts here either. I'll make the necessary changes and include a unit test file on my next commit. Will keep you posted. 
- Mukkesh